src/main/java/seedu/address/ui/CapPanel.java

```diff
@@ -53,7 +53,9 @@ public void handleTranscriptChangedEvent(TranscriptChangedEvent event) {
         ReadOnlyTranscript transcript = event.data;
         Platform.runLater(() -> currentCapDouble.setValue(transcript.getCurrentCap()));
         CapGoal goal = transcript.getCapGoal();
-        Platform.runLater(() -> capGoalString.setValue(goal.isSet() ? String.valueOf(goal.getValue()) : "NIL"));
+        Platform.runLater(() -> capGoalString.setValue(
```

Comment:
Try not to use ternary operator in another ternary operator or at least make it a bit more readable